libraries/src/Helper/MediaHelper.php

```diff
@@ -185,7 +185,7 @@ public static function checkFileExtension($extension, $component = 'com_media',
         $allowable = array_map('trim', explode(',', $params->get('restrict_uploads_extensions', 'bmp,gif,jpg,jpeg,png,webp,avif,ico,mp3,m4a,mp4a,ogg,mp4,mp4v,mpeg,mov,odg,odp,ods,odt,pdf,ppt,txt,xcf,xls,csv')));
         $ignored   = array_map('trim', explode(',', $params->get('ignore_extensions', '')));
 
-        if ($extension == '' || $extension == false || (!\in_array($extension, $allowable, true) && !\in_array($extension, $ignored, true))) {
+        if ($extension == '' || !$extension || (!\in_array($extension, $allowable, true) && !\in_array($extension, $ignored, true))) {
```

Comment:
Better remove $extension == false check
```suggestion
       if ($extension === '' || (!\in_array($extension, $allowable, true) && !\in_array($extension, $ignored, true))) {
```